common/artifacts/906_wc_artifacts_ceremonial_torso.txt

```diff
@@ -24,6 +24,10 @@ tabard_of_the_silverhand = {
 		is_adult = yes
 		religion = holy_light
 	}
+	allowed_gift = {
+		religion_group = light_group
+	}
```

Comment:
Don't get it. Only `religion = holy_light` can wear it.

Reply:
sry, my bad, will change to holy_light

Reply:
ok, changed